### DISCUSSION TITLE
[osu! lazer] Can Nominators, Genre, Language be added in the details?

### DISCUSSION THREAD

**Discussion by juices95** [ppy/osu#23127] [Ideas]
osu!(lazer)
![imag010e](https://user-images.githubusercontent.com/122288984/229797672-4504380b-c5e1-4b02-9b63-0cb724aa4971.png)

osu!(web)
![image02](https://user-images.githubusercontent.com/122288984/229797864-74900fb5-d5c5-4d30-8160-96166bfdccd1.png)

**Comment by Rekunan** [comment]
This is planned in the song select redesign: https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0-1

**Reply by juices95** [reply]
Nominators may also consider added.  thx

**Comment by juices95** [comment]
In addition, the name of Guest Participation Beatmaps can also be added~

![image](https://user-images.githubusercontent.com/122288984/229861943-95d737aa-3dae-4e73-a9c3-d53037ebb6e8.png)
